pulsar-broker/src/test/java/org/apache/pulsar/broker/systopic/PartitionedSystemTopicTest.java

```diff
@@ -235,4 +235,27 @@ private void testSetBacklogCausedCreatingProducerFailure() throws Exception {
             Assert.fail("failed to create producer");
         }
     }
+
+    @Test
+    public void testGetTopic() throws Exception {
```

Comment:
One question about the test.
Why the test should be added to the `PartitionedSystemTopicTest.java`?
Looks like they are unrelated.

The `BrokerServiceTest` looks more reasonable

Reply:
ok

Reply:
fixed.